smart-contracts/contracts/Unlock.sol

```diff
@@ -81,8 +81,8 @@ contract Unlock is UnlockInitializable, UnlockOwnable {
   // The WETH token address, used for value calculations
   address public weth;
 
-  // The UDT token address, used to mint tokens on referral
-  address public udt;
```

Comment:
We should probably keep that and mark it as deprecated in favor of `governanceToken` to avoid breaking 3rd parties?